tests/cross_functional/conftest.py

```diff
@@ -1346,6 +1346,6 @@ def factory(
         )
         feature_setup_map["executor"]["event"] = event
         feature_setup_map["executor"]["threads"] = threads
-        return feature_setup_map
+        return feature_setup_map, all_buckets
```

Comment:
this change will break the existing system tests @PrasadDesala because of unpacking.

Reply:
instead you can add all_buckets to the feature_setup_map dict with key "all_buckets"

Reply:
Done